frontend/src/bundles/auth/components/sign-up-form/sign-up-form.tsx

```diff
@@ -24,26 +55,82 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }) => {
         [handleSubmit, onSubmit],
     );
 
+    const renderRadioGroup = useCallback(function renderRadio({
+        field,
+    }: {
+        field: ControllerRenderProps<UserSignUpRequestDto, 'role'>;
+        fieldState: ControllerFieldState;
+        formState: UseFormStateReturn<UserSignUpRequestDto>;
+    }): React.ReactElement {
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
```suggestion
```

Reply:
If I don't destructure out ref:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/90120572/a1cc0841-f8fc-4c76-8818-d9cc338b4e9c)


I get this error:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/90120572/b661780d-49cb-4c70-b0c0-6197232bd3a7)

Reply:
why do we use Controler? Can we validate the form without this component?

Reply:
I am currently researching this, because all examples I see use a variant of this:
```
<FormControl>
         <TextField label="Email *" {...register('email', { required: true})} name='email'/>
</FormControl>
```

And it should work in theory, but for now I can't get the form to validate. Will come back with more info on this.

Reply:
Bugfix bt-10 has a working radio-group that removes the need to use `Controller`.